src/apps/reservation-list/reservation-material/reservation-status.tsx

```diff
@@ -22,16 +24,22 @@ const ReservationStatus: FC<ReservationStatusProps> = ({
   empty = false,
   infoLabel,
   label,
-  children
+  children,
+  showArrow = true,
+  classNameOverride
 }) => {
   const notificationClickEventHandler = useCallback(() => {
     if (openReservationDetailsModal && reservationInfo) {
       openReservationDetailsModal(reservationInfo);
     }
   }, [openReservationDetailsModal, reservationInfo]);
 
+  const className =
+    classNameOverride !== undefined
+      ? classNameOverride
+      : "list-reservation__status";
   return (
-    <div className="list-reservation__status">
+    <div className={className}>
```

Comment:
Consider whether `classNameOverride` could be renamed to `className` with a default value of `list-reservation__status`.

This seems like a pattern you are using elsewhere e.g. `showArrow`.